### PR TITLE
VIH-8896 tracking datetime columns

### PR DESCRIPTION
**Pull request by muralikaturi**
**Before creating a pull request make sure that:**

- [ x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x ] tests have been updated / new tests has been added (if needed)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: VideoApi/VideoApi.DAL/Migrations/20220405223938_AddTrackingDateTimeColumns.cs [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+                name: "CreatedAt",
+                table: "Room",
+                nullable: true);
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: "UpdatedAt",
+                table: "Room",
+                nullable: true);
+
+            migrationBuilder.AlterColumn<Guid>(
```
Comment (oliver-scott) [inline VideoApi/VideoApi.DAL/Migrations/20220405223938_AddTrackingDateTimeColumns.cs:45]: Why is this needed? Do we know for sure that this value is not currently null in all databases?

File: VideoApi/VideoApi.DAL/Migrations/20220405223938_AddTrackingDateTimeColumns.cs [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+                name: "CreatedAt",
+                table: "Endpoint",
+                nullable: true);
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: "UpdatedAt",
+                table: "Endpoint",
+                nullable: true);
+
+            migrationBuilder.AlterColumn<Guid>(
```
Comment (oliver-scott) [inline VideoApi/VideoApi.DAL/Migrations/20220405223938_AddTrackingDateTimeColumns.cs:83]: as per previous comment

**Review by omio-ma — APPROVED** [review]
